FLASK/todolist_api/file_changes.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+
+def read_tasks_from_file(file_path='tasks.json'):
+    try:
+        with open(file_path, "r") as file:
+            tasks_data = json.load(file)
+        return tasks_data
+    except (FileNotFoundError, json.JSONDecodeError) as error:
+        print(f"Error reading tasks from file: {error}")
```

Comment:
Ai putea sa arunci un custom exception aici. Te rog sa citesti despre ele si cum functioneaza in Python